optimize/backend/src/it/java/io/camunda/optimize/test/it/extension/db/DatabaseTestService.java

```diff
@@ -62,7 +62,7 @@
 public abstract class DatabaseTestService {
 
   protected static final ObjectMapper OBJECT_MAPPER = createObjectMapper();
-  private static final Logger log = org.slf4j.LoggerFactory.getLogger(DatabaseTestService.class);
+  private static final Logger LOG = org.slf4j.LoggerFactory.getLogger(DatabaseTestService.class);
```

Comment:
NABD but why do we have the fully qualified method here rather than an import?

Reply:
I don't think there's a reason. Maybe it got generated by some IDE, this way.
I'm not changing it here simply because that would trigger all of the pipelines again, but possibly I can sneak the change in, in the next PR

Reply:
I think it's NABD, and the same comment probably applies to many more files. If it's easy, then feel free to do it. Equally, feel free to ignore it 👍